custom_dc/env.list

```diff
@@ -0,0 +1,34 @@
+# API key for accessing the base Data Commons API.
+DC_API_KEY=
+# API key for accessing Maps and Places APIs for exploration tools.
+MAPS_API_KEY=
+# Use local SQLite as the database.
+USE_SQLITE=true
+# Input directory containing files to be processed.
+INPUT_DIR=
+# Output directory for generated csv files.
+OUTPUT_DIR=
+# Use Google Cloud SQL as the database.
+USE_CLOUDSQL=false
+# The project ID of the Google Cloud Platform project for the custom Data Commons.
+GOOGLE_CLOUD_PROJECT=
+# The name of the Cloud SQL instance, in the form <PROJECT_ID>:<LOCATION>:<INSTANCE_ID>.     
+CLOUDSQL_INSTANCE=
+# The name of the Google Cloud SQL database.
+DB_NAME=datacommons
+# The name of a database user configured to access the Cloud SQL instance. It may be root or another user you have configured.
+DB_USER=
+# The password of the user specified in DB_USER.
+DB_PASS=
+# (Optional) If used, the Redis Memorystore instance IP address.
+REDIS_HOST=
+# The data path of the files stored in Google Cloud Storage, in the form gs://<BUCKET>/<FOLDER>/.
```

Comment:
Add a comment:
"USE GCS_DATA_ATH if OUTPTU folder is on GCS, else use OUTUT_DIR for a local output folder."
move OUTPUT_DIR and GCS_DATA_PATH together.